### PR TITLE
Fix typo: vSphere port is required.

### PR DESCRIPTION
**Pull request by lindhe**
Port is required:

image

### CODE REVIEW

**Comment by lindhe** [comment]
What's a common port number to use here?

**Comment by martyav** [comment]
This page might be helpful: https://ranchermanager.docs.rancher.com/getting-started/installation-and-upgrade/installation-requirements/port-requirements#ports-for-rancher-launched-kubernetes-clusters-using-node-pools

**Comment by lindhe** [comment]
Thanks, but I don't think so. This is the port for connecting to vCenter. In my case, it seems to be 443.

File: versioned_docs/version-2.7/reference-guides/cluster-configuration/downstream-cluster-configuration/node-template-configuration/vsphere.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Your cloud credential has these fields:
 | Credential Field | Description |
 |-----------------|--------------|
 | vCenter or ESXi Server |  Enter the vCenter or ESXi hostname/IP. ESXi is the virtualization platform where you create and run virtual machines and virtual appliances. vCenter Server is the service through which you manage multiple hosts connected in a network and pool host resources. |
-| Port | Optional: configure configure the port of the vCenter or ESXi server. |
+| Port | Configure configure the port of the vCenter or ESXi server. |
```
Comment (martyav) [inline versioned_docs/version-2.7/reference-guides/cluster-configuration/downstream-cluster-configuration/node-template-configuration/vsphere.md:16]:
```suggestion
| Port | Configure the port of the vCenter or ESXi server. |
```

**Pull request merged**